server/tools/hatohol-server-type-util

```diff
@@ -132,7 +132,7 @@ def generate_one_type(table_name, server_type, server_label, parameters,
             ' uuid=\'%s\';' % \
         (server_label, parameters, plugin_path, sql_version, PLUGIN_ENABLED,
          uuid)
-    sql = open('server-type-' + file_identifier + '.sql', 'w')
+    sql = open('90-server-type-' + file_identifier + '.sql', 'w')
```

Comment:
This is just an impression.
You can ignore it.

We should recommend to use `90-server-type-` prefix for 3rd party HAP2 plugin developers?
Or, 3rd party HAP2 developer should insert HAP2 plugin definition into DB with another mechanism?